readme.txt

```diff
@@ -133,426 +133,7 @@ For reasons of data protection, it is not possible to see the followers of other
-* support authorized fetch to avoid having comments from "Anonymous". props [@BenLubar](https://github.com/BenLubar)
-* add new post type: "title and link only". props [@bgcarlisle](https://github.com/bgcarlisle)
-
-= 0.9.1 =
-
-* disable shared inbox
-* disable delete activity
-
-= 0.9.0 =
-
-* some code refactorings
-* fix #73
-
-= 0.8.3 =
-
-* fixed accept header bug
-
-= 0.8.2 =
-
-* add all required accept header
-* better/simpler accept-header handling
-* add debugging mechanism
-* Add setting to enable AP for different (public) Post-Types
-* explicit use of global functions
-
-= 0.8.1 =
-
-* fixed PHP warnings
-
-= 0.8.0 =
-
-* Moved followers list to user-menu
-
-= 0.7.4 =
-
-* added admin_email to metadata, to be able to "Manage your instance" on https://fediverse.network/manage/
-
-= 0.7.3 =
-
-* refactorings
-* fixed PHP warnings
-* better hashtag regex
-
-= 0.7.2 =
-
-* fixed JSON representation of posts https://merveilles.town/@xuv/101907542498716956
-
-= 0.7.1 =
-
-* fixed inbox problems with pleroma
-
-= 0.7.0 =
-
-* finally fixed pleroma compatibility
-* added "following" endpoint
-* simplified "followers" endpoint
-* fixed default value problem
-
-= 0.6.0 =
-
-* add tags as hashtags to the end of each activity
-* fixed pleroma following issue
-* followers-list improvements
-
-= 0.5.1 =
-
-* fixed name-collision that caused an infinite loop
-
-= 0.5.0 =
-
-* complete refactoring
-* fixed bug #30: Password-protected posts are federated
-* only send Activites when ActivityPub is enabled for this post-type
-
-= 0.4.4 =
-
-* show avatars
-
-= 0.4.3 =
-
-* finally fixed backlink in excerpt/summary posts
-
-= 0.4.2 =
-
-* fixed backlink in excerpt/summary posts (thanks @depone)
-
-= 0.4.1 =
-
-* finally fixed contact list
-
-= 0.4.0 =
-
-* added settings to enable/disable hashtag support
-* fixed follower list
-* send activities only for new posts, otherwise send updates
-
-= 0.3.2 =
-
-* added "followers" endpoint
-* change activity content from blog 'excerpt' to blog 'content'
-
-= 0.3.1 =
-
-* better json encoding
-
-= 0.3.0 =
-
-* basic hashtag support
-* temporarily deactivated likes and boosts
-* added support for actor objects
-* fixed encoding issue
-
-= 0.2.1 =
-
-* customizable backlink (permalink or shorturl)
-* show profile-identifiers also on profile settings
-
-= 0.2.0 =
-
-* added option to switch between content and excerpt
-* removed html and duplicate new-lines
-
-= 0.1.1 =
-
-* fixed "excerpt" in AS JSON
-* added settings for the activity-summary and for the activity-object-type
-
-= 0.1.0 =
-
-* added basic WebFinger support
-* added basic NodeInfo support
-* fully functional "follow" activity
-* send new posts to your followers
-* receive comments from your followers
-
-= 0.0.2 =
-
-* refactoring
-* functional inbox
-* nicer profile views
-
-= 0.0.1 =
-
-* initial
+See Changelog on [GitHub](https://github.com/Automattic/wordpress-activitypub/blob/master/CHANGELOG.md).
```

Comment:
I would suggest keeping the last major release and its minor updates in `readme.txt`, so folks can see the changes in their WordPress dashboard without having to click through.